spec/worker-helpers-spec.js

```diff
@@ -61,9 +61,10 @@ describe('Worker Helpers', () => {
   })
 
   describe('getESLintInstance && getESLintFromDirectory', () => {
+    const pathPart = Path.join('testing', 'eslint', 'node_modules')
```

Comment:
Why do we need to break out a separate variable for this?  It adds some indirection and complexity, and I'm not sure I see the benefit.

Reply:
The lines below where this chunk is used were over the 100 char limit otherwise, and the previous method of breaking it into multiple lines doesn't fit the current style guide. I'm open to alternatives though!